release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -231,6 +282,184 @@ You must use the `oc create secret` command instead.
+* `crd_openapi_controller_longest_running_processor_microseconds`
+* `DiscoveryController_adds`
+* `crd_autoregistration_controller_longest_running_processor_microseconds`
+* `autoregister_unfinished_work_seconds`
+* `crd_naming_condition_controller_queue_latency`
+* `crd_naming_condition_controller_retries`
+* `non_structural_schema_condition_controller_queue_latency`
+* `crd_naming_condition_controller_depth`
+* `AvailableConditionController_longest_running_processor_microseconds`
+* `crdEstablishing_depth`
+* `crd_finalizer_longest_running_processor_microseconds`
+* `crd_naming_condition_controller_adds`
+* `APIServiceOpenAPIAggregationControllerQueue1_longest_running_processor_microseconds`
+* `DiscoveryController_queue_latency`
+* `DiscoveryController_unfinished_work_seconds`
+* `crd_openapi_controller_depth`
+* `APIServiceOpenAPIAggregationControllerQueue1_queue_latency`
+* `APIServiceOpenAPIAggregationControllerQueue1_unfinished_work_seconds`
+* `DiscoveryController_work_duration`
+* `autoregister_adds`
+* `crd_autoregistration_controller_queue_latency`
+* `crd_finalizer_retries`
+* `AvailableConditionController_unfinished_work_seconds`
+* `autoregister_longest_running_processor_microseconds`
+* `non_structural_schema_condition_controller_unfinished_work_seconds`
+* `APIServiceOpenAPIAggregationControllerQueue1_depth`
+* `AvailableConditionController_depth`
+* `DiscoveryController_retries`
+* `admission_quota_controller_depth`
+* `crdEstablishing_adds`
+* `APIServiceOpenAPIAggregationControllerQueue1_retries`
+* `crdEstablishing_queue_latency`
+* `non_structural_schema_condition_controller_longest_running_processor_microseconds`
+* `autoregister_work_duration`
+* `crd_openapi_controller_retries`
+* `APIServiceRegistrationController_work_duration`
+* `crdEstablishing_work_duration`
+* `crd_finalizer_adds`
+* `crd_finalizer_depth`
+* `crd_openapi_controller_queue_latency`
+* `APIServiceOpenAPIAggregationControllerQueue1_work_duration`
+* `APIServiceRegistrationController_queue_latency`
+* `crd_autoregistration_controller_depth`
+* `AvailableConditionController_queue_latency`
+* `admission_quota_controller_queue_latency`
+* `crd_naming_condition_controller_work_duration`
+* `crd_openapi_controller_work_duration`
+* `DiscoveryController_depth`
+* `crd_naming_condition_controller_longest_running_processor_microseconds`
+* `APIServiceRegistrationController_depth`
+* `APIServiceRegistrationController_longest_running_processor_microseconds`
+* `crd_finalizer_unfinished_work_seconds`
+* `crdEstablishing_retries`
+* `admission_quota_controller_unfinished_work_seconds`
+* `non_structural_schema_condition_controller_adds`
+* `APIServiceRegistrationController_unfinished_work_seconds`
+* `admission_quota_controller_work_duration`
+* `autoregister_depth`
+* `autoregister_retries`
+* `kubeproxy_sync_proxy_rules_latency_microseconds`
+* `rest_client_request_latency_seconds`
+* `non_structural_schema_condition_controller_retries`
+
```

Comment:
We need to also add:
> Drop high granularity request duration buckets - `apiserver_request_duration_seconds_bucket`, leaving enough buckets for meaningful alerting, but drastically reducing cardinality. (38 -> 13 buckets)

Reply:
@lilic Are you recommending `apiserver_request_duration_seconds_bucket` be added to the *API server metrics* subsection for removed Kubernetes metrics? Or should this be a separate section (possibly deprecation?) outlining this particular metric and why it should be dropped?

Reply:
I guess a separate section makes sense, thanks!

Reply:
@lilic I added this info via this commit: 6fa3cbd4c313a98d5fa44d76520b17f7b3c592b6. I tried giving a bit more context (found [MON-847](https://issues.redhat.com/browse/MON-847), which is where I dug up some more info), but will definitely need you to review it. Let me know if I'm way off track. Thanks!